tests/progs/test_jubjub.py

```diff
@@ -216,3 +212,28 @@ def test_basic_point_functionality():
             await p1.montgomery_mul(5))
 
     await run_test_prog(_prog, test_preprocessing)
+
+
+@mark.asyncio
+async def test_share_mul(test_preprocessing):
+
+    p = TEST_POINTS[1]
+    multiplier = 7
+    multiplier_bin = bin(multiplier)[2:]
+
+    async def _prog(context):
+        print("type(SharedIdeal(TEST_CURVE)): ", type(SharedIdeal(TEST_CURVE)))
```

Comment:
take this out

Also, you should check out the program [pdb](https://realpython.com/python-debugging-pdb/) for quickly getting this info-- you can invoke it by adding `--pdb` to however you run the tests (e.g. `docker-compose run --rm honeybadgermpc pytest -v --pdb`). You could put a breakpoint somewhere in the code by adding `import pdb; pdb.set_trace()` and then you can evaluate simple expressions like that to quickly test stuff out.

Reply:
Nice skill!